authors/apps/authentication/views.py

```diff
@@ -26,7 +29,20 @@ def post(self, request):
         serializer.is_valid(raise_exception=True)
         serializer.save()
 
-        return Response(serializer.data, status=status.HTTP_201_CREATED)
+        token_expiry = datetime.now() + timedelta(days=60)
+        token = jwt.encode({
+            'username': user['username'],
+            'exp': token_expiry.strftime('%s')},
+            key=settings.SECRET_KEY,
+            algorithm='HS256')
+        
+        # serializer.data['token'] = token
+        user_token = token.decode('utf-8')
+
+        new_data={'username': user['username'], 'token': user_token}
```

Comment:
@BarnaTB You change give this variable a more descriptive name  new_data.
```suggestion
        response_data={'username': user['username'], 'token': user_token}
```

Reply:
Noted. Will do that ASAP. Thanks